src/components/pager/css/Pager.css

```diff
@@ -18,41 +18,57 @@
 /* Pager UI */
 /* Class name for the list container */
 .fl-pager .fl-pager-ui {
-    margin:0; padding:0;
+    margin: 0;
+    padding: 0;
 }
 .fl-pager .fl-pager-ui li {
-    list-style-type:none;
-    display:inline;
+    list-style-type: none;
+    display: inline;
 }
 .fl-pager .fl-pager-ui a {
-    padding:0 3px;
-    margin:0 2px;
+    margin: 0 2px;
 }
 .fl-pager .fl-pager-ui .fl-pager-links a {
-    border:1px solid #ccc;
-    background-color:#fff;
-    zoom:1;
-}
-.fl-pager .fl-pager-ui .fl-pager-disabled,
-.fl-pager .fl-pager-ui .fl-pager-links .fl-pager-currentPage {
-    color:#000;
-    border:none;
-    background-color:transparent;
-    text-decoration:none;
+    border: 1px solid #ccc;
+    background-color: #fff;
+    zoom: 1;
 }
 .fl-pager .fl-pager-ui .fl-pager-links a:hover {
-    background-color:#0287C5;
-    color:#fff;
+    background-color: #0287C5;
+    color: #fff;
     text-decoration:none;
 }
+.fl-pager .fl-pager-ui .fl-pager-links .fl-pager-currentPage {
+    color: #fff;
+    border: none;
+    outline: 0px;
+    background-color: #000;
+    text-decoration: none;
+    cursor: default;
+}
+.fl-pager .fl-pager-ui .fl-pager-links .fl-pager-currentPage:hover,
+.fl-pager .fl-pager-ui .fl-pager-links .fl-pager-currentPage:focus {
+    color: #fff;
+    border: none;
+    outline: 2px #000 solid;
+    background-color: #000;
+    text-decoration: none;
+    cursor: default;
```

Comment:
is it possible to just have the outline specified here. The rest seem like duplicates from the previous style declaration.